.husky/commit-msg

```diff
@@ -1 +0,0 @@
-pnpm commitlint --edit "$1"
```

Comment:
You can do a `git commit --no-verify` if you like to skip the commit message format check^^